pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -301,6 +317,27 @@ func (f *framework) RunFilterPlugins(pc *PluginContext,
 	return nil
 }
 
+// RunPostFilterPlugins runs the set of configured post-filter plugins.
+func (f *framework) RunPostFilterPlugins(
+	pc *PluginContext,
+	pod *v1.Pod,
+	nodes []*v1.Node,
+	failedFilters FailedFilterMap,
+) *Status {
+	for _, pl := range f.postFilterPlugins {
+		status := pl.PostFilter(pc, pod, nodes, failedFilters)
+		if status.IsSuccess() {
+			continue
```

Comment:
nit: Please prefer to not use `continue` where possible. You can put the rest of the body of the loop under `if !status.IsSuccess()`.